consensus/dman/actors_test.go

```diff
@@ -74,9 +76,15 @@ type testingProxy struct {
 	skipCallCheck bool
 }
 
+// assert struct `testingProxy` implements `reqBusView` , `txMarshaller`, `databaseView` and `typeProxyIface` interfaces
```

Comment:
question: Does it matter if `testingProxy` doesn't implement these interfaces if tests cover that functionality?

Reply:
Some parts of the code require interfaces to be passed in as arguments and this ensures the `testingProxy` struct actually implements the required interfaces. Note that `testingProxy` mocks functionality to better test these components in isolation.